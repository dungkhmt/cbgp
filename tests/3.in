100 100
100 208
13 4
14 5
82 10
55 68
40 38
82 74
20 50
5 65
80 20
8 73
69 5
37 9
14 29
3 6
24 62
95 12
11 65
4 1
91 8
7 24
4 2
37 72
93 73
78 71
7 8
26 85
33 60
6 2
66 32
80 85
75 17
72 69
73 91
17 94
78 6
3 65
66 69
5 100
32 5
4 15
26 80
8 77
43 48
11 64
13 92
21 4
33 80
35 32
4 27
37 2
26 20
60 26
5 66
36 65
39 31
20 22
54 70
88 89
59 14
5 2
1 15
1 33
36 40
93 8
100 72
76 55
54 81
22 1
16 25
40 49
39 30
2 3
7 4
14 17
81 94
81 75
6 56
51 58
11 34
18 17
29 59
70 81
55 32
10 4
14 18
69 2
23 47
20 1
52 98
36 38
33 58
43 13
45 21
36 5
14 81
34 54
18 28
84 4
81 17
5 25
22 32
50 22
71 2
88 58
54 75
9 72
51 10
63 69
63 99
24 67
83 18
20 32
31 25
71 1
31 16
97 3
12 13
6 96
67 86
96 56
51 88
76 68
92 21
18 19
19 47
54 57
10 74
75 14
14 34
2 96
83 14
33 26
4 58
4 12
60 80
43 21
5 61
4 89
13 95
17 29
13 21
6 97
14 90
5 38
39 5
3 96
14 54
20 35
22 71
41 36
14 11
69 87
79 30
81 29
4 33
14 36
79 25
44 52
75 64
3 78
2 9
32 61
78 2
100 69
63 24
64 54
51 89
78 36
21 27
5 9
15 2
30 36
71 32
77 73
32 78
4 69
36 11
35 44
16 5
24 8
78 97
35 46
20 33
36 76
36 42
84 45
92 12
66 61
79 31
4 24
2 65
32 46
51 4
90 54
34 64
3 36
21 12
63 46
2 1
11 5
43 92
4 88
23 19
30 5
25 30
2 72
53 51
56 2
