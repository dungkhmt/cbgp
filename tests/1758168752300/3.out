(12, 3), (13, 4), (81, 9), (54, 67), (39, 37), (81, 73), (19, 49), (4, 64), (79, 19), (7, 72), (68, 4), (36, 8), (13, 28), (2, 5), (23, 61), (94, 11), (10, 64), (3, 0), (90, 7), (6, 23), (3, 1), (36, 71), (92, 72), (77, 70), (6, 7), (25, 84), (32, 59), (5, 1), (65, 31), (79, 84), (74, 16), (71, 68), (72, 90), (16, 93), (77, 5), (2, 64), (65, 68), (4, 99), (31, 4), (3, 14), (25, 79), (7, 76), (42, 47), (10, 63), (12, 91), (20, 3), (32, 79), (34, 31), (3, 26), (36, 1), (25, 19), (59, 25), (4, 65), (35, 64), (38, 30), (19, 21), (53, 69), (87, 88), (58, 13), (4, 1), (0, 14), (0, 32), (35, 39), (92, 7), (99, 71), (75, 54), (53, 80), (21, 0), (15, 24), (39, 48), (38, 29), (1, 2), (6, 3), (13, 16), (80, 93), (80, 74), (5, 55), (50, 57), (10, 33), (17, 16), (28, 58), (69, 80), (54, 31), (9, 3), (13, 17), (68, 1), (22, 46), (19, 0), (51, 97), (35, 37), (32, 57), (42, 12), (44, 20), (35, 4), (13, 80), (33, 53), (17, 27), (83, 3), (80, 16), (4, 24), (21, 31), (49, 21), (70, 1), (87, 57), (53, 74), (8, 71), (50, 9), (62, 68), (62, 98), (23, 66), (82, 17), (19, 31), (30, 24), (70, 0), (30, 15), (96, 2), (11, 12), (5, 95), (66, 85), (95, 55), (50, 87), (75, 67), (91, 20), (17, 18), (18, 46), (53, 56), (9, 73), (74, 13), (13, 33), (1, 95), (82, 13), (32, 25), (3, 57), (3, 11), (59, 79), (42, 20), (4, 60), (3, 88), (12, 94), (16, 28), (12, 20), (5, 96), (13, 89), (4, 37), (38, 4), (2, 95), (13, 53), (19, 34), (21, 70), (40, 35), (13, 10), (68, 86), (78, 29), (80, 28), (3, 32), (13, 35), (78, 24), (43, 51), (74, 63), (2, 77), (1, 8), (31, 60), (77, 1), (99, 68), (62, 23), (63, 53), (50, 88), (77, 35), (20, 26), (4, 8), (14, 1), (29, 35), (70, 31), (76, 72), (31, 77), (3, 68), (35, 10), (34, 43), (15, 4), (23, 7), (77, 96), (34, 45), (19, 32), (35, 75), (35, 41), (83, 44), (91, 11), (65, 60), (78, 30), (3, 23), (1, 64), (31, 45), (50, 3), (89, 53), (33, 63), (2, 35), (20, 11), (62, 45), (1, 0), (10, 4), (42, 91), (3, 87), (22, 18), (29, 4), (24, 29), (1, 71), (52, 50), (55, 1), 
numberIntersection=0 
minAngle=0.023941784554969203 
sumAngle=100.42511064476427 
minEdgeLength=1.0 
sumEdgeLength=11816.686222773085 
minDistance=0.03146583877637763 
sumDistance=2469483.185771765 

0.0000000000 0.0239417846 100.4251106448 1.0000000000 11816.6862227731 0.0314658388 2469483.1857717650
0: (62, 40),
1: (178, 41),
2: (310, 50),
3: (66, 53),
4: (231, 223),
5: (307, 40),
6: (113, 69),
7: (128, 56),
8: (178, 81),
9: (51, 67),
10: (195, 65),
11: (103, 108),
12: (81, 72),
13: (231, 140),
14: (68, 49),
15: (244, 169),
16: (215, 91),
17: (212, 89),
18: (216, 101),
19: (0, 19),
20: (112, 73),
21: (135, 22),
22: (227, 130),
23: (104, 113),
24: (239, 156),
25: (35, 69),
26: (87, 61),
27: (214, 102),
28: (231, 92),
29: (258, 162),
30: (249, 169),
31: (341, 13),
32: (43, 57),
33: (264, 67),
34: (0, 4),
35: (309, 65),
36: (169, 61),
37: (233, 143),
38: (251, 170),
39: (236, 144),
40: (237, 152),
41: (233, 163),
42: (99, 74),
43: (3, 3),
44: (89, 64),
45: (355, 4),
46: (216, 103),
47: (99, 73),
48: (237, 153),
49: (65, 23),
50: (43, 73),
51: (7, 2),
52: (121, 400),
53: (255, 80),
54: (334, 20),
55: (301, 43),
56: (232, 127),
57: (49, 63),
58: (230, 99),
59: (36, 60),
60: (236, 226),
61: (138, 227),
62: (238, 261),
63: (233, 72),
64: (190, 62),
65: (235, 242),
66: (64, 152),
67: (327, 28),
68: (138, 54),
69: (237, 88),
70: (191, 25),
71: (158, 61),
72: (139, 73),
73: (98, 104),
74: (205, 82),
75: (296, 98),
76: (132, 63),
77: (316, 36),
78: (250, 164),
79: (29, 55),
80: (234, 90),
81: (62, 152),
82: (215, 101),
83: (82, 61),
84: (33, 60),
85: (63, 153),
86: (72, 55),
87: (50, 64),
88: (51, 65),
89: (245, 102),
90: (130, 58),
91: (99, 75),
92: (133, 66),
93: (221, 90),
94: (88, 82),
95: (302, 46),
96: (309, 41),
97: (15, 0),
98: (75, 206),
99: (157, 62),
62 40 178 41 310 50 66 53 231 223 307 40 113 69 128 56 178 81 51 67 195 65 103 108 81 72 231 140 68 49 244 169 215 91 212 89 216 101 0 19 112 73 135 22 227 130 104 113 239 156 35 69 87 61 214 102 231 92 258 162 249 169 341 13 43 57 264 67 0 4 309 65 169 61 233 143 251 170 236 144 237 152 233 163 99 74 3 3 89 64 355 4 216 103 99 73 237 153 65 23 43 73 7 2 121 400 255 80 334 20 301 43 232 127 49 63 230 99 36 60 236 226 138 227 238 261 233 72 190 62 235 242 64 152 327 28 138 54 237 88 191 25 158 61 139 73 98 104 205 82 296 98 132 63 316 36 250 164 29 55 234 90 62 152 215 101 82 61 33 60 63 153 72 55 50 64 51 65 245 102 130 58 99 75 133 66 221 90 88 82 302 46 309 41 15 0 75 206 157 62 
0: (62, 40), 1: (178, 41), 2: (310, 50), 3: (66, 53), 4: (231, 223), 5: (307, 40), 6: (113, 69), 7: (128, 56), 8: (178, 81), 9: (51, 67), 10: (195, 65), 11: (103, 108), 12: (81, 72), 13: (231, 140), 14: (68, 49), 15: (244, 169), 16: (215, 91), 17: (212, 89), 18: (216, 101), 19: (0, 19), 20: (112, 73), 21: (135, 22), 22: (227, 130), 23: (104, 113), 24: (239, 156), 25: (35, 69), 26: (87, 61), 27: (214, 102), 28: (231, 92), 29: (258, 162), 30: (249, 169), 31: (341, 13), 32: (43, 57), 33: (264, 67), 34: (0, 4), 35: (309, 65), 36: (169, 61), 37: (233, 143), 38: (251, 170), 39: (236, 144), 40: (237, 152), 41: (233, 163), 42: (99, 74), 43: (3, 3), 44: (89, 64), 45: (355, 4), 46: (216, 103), 47: (99, 73), 48: (237, 153), 49: (65, 23), 50: (43, 73), 51: (7, 2), 52: (121, 400), 53: (255, 80), 54: (334, 20), 55: (301, 43), 56: (232, 127), 57: (49, 63), 58: (230, 99), 59: (36, 60), 60: (236, 226), 61: (138, 227), 62: (238, 261), 63: (233, 72), 64: (190, 62), 65: (235, 242), 66: (64, 152), 67: (327, 28), 68: (138, 54), 69: (237, 88), 70: (191, 25), 71: (158, 61), 72: (139, 73), 73: (98, 104), 74: (205, 82), 75: (296, 98), 76: (132, 63), 77: (316, 36), 78: (250, 164), 79: (29, 55), 80: (234, 90), 81: (62, 152), 82: (215, 101), 83: (82, 61), 84: (33, 60), 85: (63, 153), 86: (72, 55), 87: (50, 64), 88: (51, 65), 89: (245, 102), 90: (130, 58), 91: (99, 75), 92: (133, 66), 93: (221, 90), 94: (88, 82), 95: (302, 46), 96: (309, 41), 97: (15, 0), 98: (75, 206), 99: (157, 62), 
numberIntersection=0 minAngle=0.023941784554969203 sumAngle=100.42511064478904 minEdgeLength=1.0 sumEdgeLength=11816.686222773118 minDistance=0.03146583877637763 sumDistance=2469483.185771794 
Time taken: 56.353 s
